### ISSUE TITLE
Unexpected behavior with side nav in the guide while using JAWS

### ISSUE THREAD

**Issue by kriskristin** [cityofaustin/techstack#3291]
**Describe the bug**
When using JAWS in the Guide on the side nav, if the user clicks on an element (JAWS command = enter) they are not brought to the main content of that section. Visually the focus jumps, but the screen reader stays on the left nav.

**More Details**
To ensure accessibility, use either a device independent event handler (one that works with both the mouse and the keyboard) or use both mouse dependent and keyboard dependent event handlers. Get more info [here](https://webaim.org/techniques/javascript/eventhandlers)

**URL**
https://janis.austintexas.io/en/health-safety/healthcare-prevention/disease-prevention/mobile-food-vendor-permit-guide/#Learn-and-prepare-1

**See video (will add later)**

**Current behavior**
While using JAWS when an element is clicked in the side nav label, the user still remains in the side navigation.

**Expected behavior**
While using JAWS when an element is clicked in the side nav label, the user should be brought to the corresponding content in the main section of the guide. 

**Device**
Desktop JAWS

**Browser**
IE

**Additional context**
See documentation of the requirements with onClick and JAWS [here](https://webaim.org/techniques/javascript/eventhandlers) and [here](https://webaim.org/techniques/javascript/eventhandlers)

**Screenshot of the potential issue**

![Screen Shot 2019-10-28 at 9 09 37 PM](https://user-images.githubusercontent.com/12438022/67808830-6c78ae80-fa65-11e9-9710-9de5960b4784.png)

**Comment by chiaberry** [comment]
https://janis-3291-jaws-guide.netlify.com/es/health-safety/healthcare-prevention/disease-prevention/mobile-food-vendor-permit-guide/#Apply-for-a-permit-1 for QA

**Comment by desigonz** [comment]
Sent an email today for David Ondich to verify. Cc'd you on it!

**Comment by desigonz** [comment]
Hi @chiaberry—I followed up with David, let's see if we hear anything. For now, I think we can consider QA complete on this and open a new issue if it arises.

**Comment by desigonz** [comment]
Actually assigning @anukandimalla to verify at the library
